packages/app-store/routing-forms/trpc/utils.ts

```diff
@@ -170,7 +170,11 @@ export async function _onFormSubmission(
           "triggerFormSubmittedNoEventWebhook",
           {
             responseId,
-            form,
+            form: {
```

Comment:
This should be the common pattern, to ensure no data is added accidentally, but this is enough for now.